lib/main_containers.pm

```diff
@@ -95,27 +95,31 @@ sub load_host_tests_podman {
 }
 
 sub load_host_tests_docker {
+    loadtest 'microos/install_docker' if is_sle_micro;
     load_container_engine_test('docker');
     load_image_test('docker');
     load_3rd_party_image_test('docker');
     loadtest 'containers/docker_firewall' unless is_openstack;
     unless (is_sle("<=15") && is_aarch64) {
         # these 2 packages are not avaiable for <=15 (aarch64 only)
-        # zypper-docker is not available in factory
-        loadtest 'containers/zypper_docker' unless is_tumbleweed;
+        # zypper-docker is not available in factory and in SLE Micro
+        loadtest 'containers/zypper_docker' unless (is_tumbleweed || is_sle_micro);
         loadtest 'containers/docker_runc';
     }
-    unless (check_var('BETA', 1)) {
+    unless (check_var('BETA', 1) || is_sle_micro) {
         # These tests use packages from Package Hub, so they are applicable
         # to maintenance jobs or new products after Beta release
+        # PackageHub is not available in SLE Micro
         loadtest 'containers/registry' if is_x86_64;
         loadtest 'containers/docker_compose';
     }
     # works currently only for x86_64, more are coming (poo#103977)
     # Expected to work for all but JeOS on 15sp4 after
     # https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13860
     # Disabled on svirt backends (VMWare, Hyper-V and XEN) as the device name might be different than vdX
-    loadtest 'containers/validate_btrfs' if (is_x86_64 and is_qemu and !is_openstack);
+    if ((is_x86_64 && is_qemu) && !(is_openstack || is_sle_micro)) {
```

Comment:
Why?

Reply:
It requires an additional disk, we already have HDD_1 and HDD_2 for SLEM/MicroOS.. We are already running this test in general TW/SLE container host tests... I don't think it's worth it here.

Reply:
The additional disk can be solved with `NUMDISKS=3`, unfortunately the test has hardcoded it will use the second one (vdb). I guess, we can work on that later if there is an interest to include this module in MicroOS/SLEM testing. WDYT?

Reply:
Yes, that was another reason I decided to omit it for now (it was never run anyways).

Reply:
I guess usage of this https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/btrfs_test.pm#L22 would help in the future to make the test more versatile